### PR TITLE
IME-127 (FxpConversionErrorModal Fix)

### PR DESCRIPTION
**Pull request by mwan9ilwa**
- Fixed the TransfersError Modal, Transfer Details Modal not working and showing undefined values.
- Fixed RowClick not working in FxpConversionErrorModal
- Fixed Conversions Details Modal
- To be done: Mapping the correct endpoint, reducers and actions for Conversions Details Modal

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
-SKIP_PREFLIGHT_CHECK=true
-PROXY_TARGET=http://senderfsp-pm4ml.qa.prod-dev.mbox-dev.io:20000
-PROXY_PORT=10000
-PROXY_API_PATH=/experience-api
-PORT=8081
+SKIP_PREFLIGHT_CHECK=true
+PROXY_TARGET=http://senderfsp-pm4ml.qa.prod-dev.mbox-dev.io:20000
+PROXY_PORT=10000
+PROXY_API_PATH=/experience-api
+PORT=5500
```
Comment (vijayg10) [inline .env:5]: Please do not change the default values there. It breaks the local development process for somebody else.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (vijayg10) [inline .vscode/launch.json:15]: We don't usually commit vs code config files.

File: data.json [hunk 3]
```diff
@@ -0,0 +1,360 @@
+{
```
Comment (vijayg10) [inline data.json:1]: If this file is for testing, please move it to test folder

File: on-premise-deploy/docker-compose/docker-compose.yaml [hunk 4]
```diff
@@ -1,295 +1,295 @@
...
+#  │            │◄───┤ Connector  │◄──┤             │◄───┤          │
+#  └────────────┘    └────────────┘   └─────────────┘    └──────────┘
+#
+
+services:
+
+  frontend:
+    build: ../../
+    environment:
+      - API_BASE_URL=http://localhost:4010/
```
Comment (vijayg10) [inline on-premise-deploy/docker-compose/docker-compose.yaml:18]: Remove trailing slash here. We don't use trailing slashes in base URLs.

File: .vscode/launch.json (hunk 2 above)
Comment (mwan9ilwa) [reply .vscode/launch.json:15]: Weren't this supposed to be included in the gitignore at beginning of the project to automatically exclude this config from getting committed and pushed?

File: .env (hunk 1 above)
Comment (mwan9ilwa) [reply .env:5]: can I add .env to gitignore so that my local configs doesn't get pushed?

Comment (vijayg10) [reply .env:5]: Yes I think that's the way to go

File: .vscode/launch.json (hunk 2 above)
Comment (vijayg10) [reply .vscode/launch.json:15]: Yes please include in gitgnore

Comment (mwan9ilwa) [reply .vscode/launch.json:15]: alright cool

**Pull request closed without merging**